db/seeds.rb

```diff
@@ -17,6 +39,19 @@
 User.create(email: 'admin@gmail.com', full_name: "admin_user", display_name: "admin_user", password: "test", role: 1)
 
 #Create Orders
+
+30.times do
+  selected_item = Item.all.sample
+  selected_user= User.all.sample
+  random_status = [1, 2, 3, 4].sample
+  Order.create(user_id: selected_user.id,
+              status_id: random_status,
+              created_at: Faker::Time.between(2.days.ago, Time.now),
+              updated_at: Faker::Time.between(2.days.ago, Time.now) 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.